src/test/java/nl/cherement/jak/service/CardServiceTests.java

```diff
@@ -31,21 +28,16 @@
 
     @BeforeEach
     public void initialize() {
-        card = new CardEntity();
-        card2 = new CardEntity();
-        cards = new ArrayList<CardEntity>();
-
-        card.setId(1l);
-        card2.setId(1l);
+        CardEntity card = new CardEntity();
```

Comment:
de naam en de declaratie is hier op 1 line maar overal is het opgesplitst in 2, zelfs met cards in hetzelfde file.

Reply:
cards word in andere methodes gebruikt en is daarom een global variable. card wordt alleen in deze methode geburikt dus kan local zijn.